client/dashboard/sites/site-badge/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Badge } from '@automattic/ui';
+import { __ } from '@wordpress/i18n';
+
+export type EnvironmentType = 'production' | 'staging';
+
+interface SiteBadgeProps {
+	environmentType: EnvironmentType;
+}
+
+const SiteBadge = ( { environmentType }: SiteBadgeProps ) => {
+	const getBadgeColor = (): string => {
+		return environmentType === 'staging' ? '#f0c930' : '#dcdcde';
+	};
+
+	const getBadgeText = (): string => {
+		return environmentType === 'staging' ? __( 'Staging' ) : __( 'Production' );
+	};
+
+	return <Badge style={ { backgroundColor: getBadgeColor() } }>{ getBadgeText() }</Badge>;
+};
+
+export default SiteBadge;
```

Comment:
I am curious to hear what you think about introducing a dedicated SiteBadge component that could be reused throughout the dashboard.

Reply:
> (I'd put it in client/dashboard/sites/ for now)

> Do you mean moving this one to a global components folder? I think that's a great idea, I would do it once it needs to be reused.

Yes, I am also curious if you maybe meant `client/dashboard/components/`? 🤔 As currently it is in `client/dashboard/sites/site-badge/` (which I am currently changing to `client/dashboard/sites/site-environment-badge/`)

Reply:
> Yes, I am also curious if you maybe meant client/dashboard/components/?

Works!